tests/common.go

```diff
@@ -432,6 +437,15 @@ func ParseFlags() {
 			}
 		})
 	}
+
+	// Set logging
+	logLvl, err := logrus.ParseLevel(instance.LogLevel)
+	if err != nil {
+		logrus.Fatalf("Failed to set log level due to Err: %v", err)
+	}
+	logrus.SetLevel(logLvl)
+	logrus.StandardLogger().Hooks.Add(log.NewHook())
```

Comment:
you don't need to do this again here. It's already being done in init(). Just set the log level here.

Reply:
Also did you test if this actually works? i.e invoke with different log levels in a private job.

Reply:
Yes I have tested this with a private image, it works.